examples/Microsoft.Spark.CSharp.Examples/Sql/VectorUdfs.cs

```diff
@@ -3,11 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using Apache.Arrow;
-using Apache.Arrow.Types;
 using Microsoft.Spark.Sql;
 using Microsoft.Spark.Sql.Types;
 using StructType = Microsoft.Spark.Sql.Types.StructType;
+using FxDataFrame = Microsoft.Data.Analysis.DataFrame;
```

Comment:
Same question that @eerhardt posed: 

@imback82  @rapoth - do either of you have any thoughts/opinions on how to make this better? Using 2 types named DataFrame in Spark seems unfortunate. Is there a better term we can call the corefx DataFrame class?

Reply:
We should think about this sooner rather than later. For ex: In the last .NET Town Hall, Spark Notebooks was demo'd and Dan had a question of whether the `DataFrame` he saw in the sample was a Spark `DataFrame` or the corefx `DataFrame`.